src/header/header-buttons.scss

```diff
@@ -0,0 +1,230 @@
+  &:hover {
+    background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-6));
+  }
+
+  // Active state
+  &[aria-current] {
+    background-color: rgba(var(--iui-color-foreground-primary-rgb), var(--iui-opacity-6));
+    box-shadow: 0 $iui-xxs 0 0 currentColor;
+    color: var(--iui-color-foreground-primary);
+    pointer-events: none; // Removes hover effect, reduces specificity
+    @media (forced-colors: active) {
+      color: LinkText;
+      border-bottom: 3px solid LinkText;
+    }
+  }
+
+  // Disabled state
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+    background-color: transparent;
+    color: var(--iui-text-color-placeholder);
+    pointer-events: none; // Removes hover effect
+
+    // Disabled active state
+    &[aria-current] {
+      box-shadow: none;
+      background-color: var(--iui-color-background-2);
+
+      @media (forced-colors: active) {
+        border-bottom: 3px solid GrayText;
+      }
+    }
+  }
+}
+
+@mixin iui-header-breadcrumb-button {
+  @include iui-reset;
+  @include iui-focus;
+  display: inline-flex;
+  align-items: center;
+  vertical-align: middle;
+  justify-content: center;
+  height: 100%;
+  border-radius: 0;
+  max-width: 25vw;
+  overflow: hidden;
+  padding-left: $iui-sm;
+  padding-right: $iui-sm;
+  box-sizing: border-box;
+  font-family: inherit;
+  -webkit-tap-highlight-color: transparent;
+  transition: background-color $iui-speed-fast ease-out;
+  color: inherit;
+  gap: $iui-sm;
+  pointer-events: auto; // Reverts change from parent element
+  cursor: pointer;
+  @media (forced-colors: active) {
+    border: 1px solid transparent;
+    border-bottom-width: 0;
+
+    &:not(:last-child) {
+      border-right-width: 0;
+    }
+  }
+
+  // Default & hover state
+  &,
+  &:hover {
+    background-color: transparent;
+  }
+
+  // Disabled state
+  &:disabled,
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+  }
+}
+
+@mixin iui-header-breadcrumb-button-split {
+  padding: 0 $iui-xs;
+  flex-shrink: 0;
+
+  &:hover {
+    background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-6));
+  }
+
+  @at-root [aria-current] &:hover {
+    background-color: rgba(var(--iui-color-foreground-primary-rgb), var(--iui-opacity-6));
+  }
+
+  &:disabled,
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+
+    &:hover {
+      background-color: transparent;
+    }
+  }
+}
+
+@mixin iui-header-breadcrumb-button-dropdown-icon {
+  width: $iui-icons-default;
+  height: $iui-icons-default;
+  fill: currentColor;
+  flex-shrink: 0;
+  margin-right: -#{$iui-xs};
+
+  @at-root .iui-header-breadcrumb-button-split & {
+    margin-right: 0;
+  }
+
+  @at-root {
+    :disabled &,
+    [aria-disabled='true'] & {
+      @media (forced-colors: active) {
+        fill: GrayText;
+      }
+    }
+  }
+}
+
+@mixin iui-header-breadcrumb-button-text {
+  @include iui-label-with-sublabel;
+
+  &-sublabel {
+    @at-root .iui-slim & {
+      display: none;
```

Comment:
I think we were smoothly transitioning the height before, but now it abruptly disappears.

Reply:
Transition effect is back in place.